### PR TITLE
[KYC-82] Twilio health check now logs response headers and content

### PR DESCRIPTION
**Pull request by yreynhout-lykke**
- Twilio health check now logs response headers and content (in case of debug loglevel).
- Made stylecop happy while I was at it.

### CODE REVIEW

File: src/Apollo/TwilioHealthCheck.cs [hunk 1]
```diff
@@ -45,9 +50,24 @@ public TwilioHealthCheck(Func<HttpClient> httpClientFactory, Uri statusEndpoint)
...
+                        builder.AppendLine(
+                            $"Twilio status api {this.statusEndpoint.OriginalString} responded with {response.StatusCode} and the following headers and content:");
+                        foreach (var header in response.Headers)
+                        {
+                            builder.AppendLine(header.ToString());
+                        }
+
+                        builder.AppendLine(await response.Content.ReadAsStringAsync());
+
+                        this.logger.Log(LogLevel.Debug, builder.ToString());
```
Comment (JoeStead) [inline src/Apollo/TwilioHealthCheck.cs]: Can just use `this.logger.Debug` I think. Preference thing really, it'll do the same thing

**Comment by JoeStead** [comment]
Do we want to add logging to the other health checks?

File: src/Apollo/TwilioHealthCheck.cs [hunk 2]
```diff
@@ -45,9 +50,24 @@ public TwilioHealthCheck(Func<HttpClient> httpClientFactory, Uri statusEndpoint)
 
                 if (response.StatusCode != HttpStatusCode.OK)
                 {
+                    if (this.logger.IsEnabled(LogLevel.Debug))
+                    {
+                        var builder = new StringBuilder();
+                        builder.AppendLine(
+                            $"Twilio status api {this.statusEndpoint.OriginalString} responded with {response.StatusCode} and the following headers and content:");
+                        foreach (var header in response.Headers)
```
[the remark this answers is not in the document]
Comment (yreynhout-lykke) [reply src/Apollo/TwilioHealthCheck.cs]: Good catch - TIL - will do.

File: src/Apollo/TwilioHealthCheck.cs (hunk 1 above)
Comment (yreynhout-lykke) [reply src/Apollo/TwilioHealthCheck.cs]: Ah, that makes sense and is easier to find in code via `Find Usages` - will do.

**Comment by yreynhout-lykke** [comment]
The other health checks are `not my code` so I don't know how feasible that is.

**Comment by JoeStead** [comment]
Ahh yeah, of course. Let's just hope :)

**Comment by yreynhout-lykke** [comment]
@JoeStead @jchannon it seems that the built-in loggers report the content of the response on the health check result (and don't have any logging). Do we want to align with that strategy? Here's a (decompiled) snippet from the `IdSrvHealthCheck`:

```csharp
return new HealthCheckResult(failureStatus, string.Format("Discover endpoint is not responding with 200 OK, the current status is {0} and the content {1}", statusCode, (object) await async.Content.ReadAsStringAsync()), (Exception) null, (IReadOnlyDictionary<string, object>) null);
```

**Pull request merged**